lib/ui/painting/image.cc

```diff
@@ -28,7 +29,10 @@ void CanvasImage::RegisterNatives(tonic::DartLibraryNatives* natives) {
 
 CanvasImage::CanvasImage() {}
 
-CanvasImage::~CanvasImage() {}
+CanvasImage::~CanvasImage() {
+  SkImage* image = image_.release();
+  Threads::IO()->PostTask([image]() { image->unref(); });
```

Comment:
Ideally, the SkImage reference should be a unique pointer. We should investigate if Skia has those. In any case, lgtm on this patch.